openwisp_users/models.py

```diff
@@ -59,6 +60,14 @@ def organizations_pk(self):
                     .values_list('organization_id')
         return qs
 
+    def clean(self):
+        if self.email:
+            try:
+                User.objects.exclude(pk=self.pk).get(email=self.email)
```

Comment:
use `User.objects.exclude(pk=self.pk).filter(email=self.email).exists()` which is going to be more efficient.

It could be:

```python
if User.objects.exclude(pk=self.pk).filter(email=self.email).exists():
    raise ValidationError({'email': 'A user with this email already exist'})
```